### DISCUSSION TITLE
如何将幕布的笔记导入到remnote

### DISCUSSION THREAD

**Discussion by thekingofall** [5eagull/RemNote-Chinese-chat-group#32] [你问我答/互相帮助]
最近对remnote产生了兴趣，然后将一些幕布的笔记迁移到了remnote。发现如果直接复制粘贴的话，会每一条rem都会多出一个上一级为空的rem，一个个删除的话太麻烦了，有人知道批量怎么搞吗？

**Comment by makerjackie** [comment]
你 ctrl shift + V 就不会多出空行

**Comment by ghost** [comment]
可以用quicker，从剪贴板去除空行自动贴到RN

**Comment by Deng-Jiajun** [comment]
看了一下幕布复制出来的内容，除了文本内容，好像唯一还剩下的就是注释，前面会带一个"｜"（不是|）
可以配合自定义的标签样式（比如##comment 或者 [##Block-Quote](https://getquicker.net/sharedaction?code=0fb2e95b-7345-446b-b00d-08d917c85b4c)）把这个｜替换为##comment，这样这些注释的内容就会显示comment的样式效果。
